src/components/Note/Note.js

```diff
@@ -104,6 +104,14 @@ const Note = ({ annotation }) => {
 
   const showReplyArea = !Object.values(isEditingMap).some(val => val);
 
+  const handleNoteKeydown = e => {
+    // Stop enter/space key on Note from being submitted into field.
+    const isNote = e.target === e.currentTarget;
+    if (isNote && (e.key === 'Enter' || e.key === ' ')) {
+      e.preventDefault();
+    }
```

Comment:
Checks if target issuing key events is the same as Note (`currentTarget`), if it is prevent <kbd>return</kbd> and <kbd>space</kbd> from doing default behaviour since upon opening the Note, this would be entered into the input field for commenting